cg/constants/delivery.py

```diff
@@ -0,0 +1,80 @@
+"""Constants for delivery"""
+from pathlib import Path
+
+PROJECT_BASE_PATH = Path("/home/proj/production/customers")
```

Comment:
It could be worth to consider where this value is stored. We are currently storing panel_bed paths in servers config, and this one refers to a path in prod folder on hasta. Perhaps this could be a path both on prod and stage (for testing) and kept in servers repo instead?

Reply:
Hmm yes I think that would be the best solution. Could we add that in another PR?

Reply:
Or I can see if it is possible to add now

Reply:
Added